USFMToolsSharp.Renderers.Docx/DocxRenderer.cs

```diff
@@ -523,7 +523,7 @@ public void createBookHeaders(string bookname)
             }
 
             // Book name
-            run.AddNewT().Value = bookname;
+            run.AddNewT().Value = bookname == null ? "" : bookname;
```

Comment:
You can do bookname ?? ""

Reply:
Oh cool, I didn't know about that syntax.  Thanks!